content/en/docs/reference/kubernetes-api/ref/common-parameters/common-parameters-.md

```diff
@@ -0,0 +1,170 @@
+---
+api_metadata:
+  apiVersion: ""
```

Comment:
As noted, do you need the `Common Parameters` directory if there is only one file?